src/chainstate/burn/db/sortdb.rs

```diff
@@ -699,6 +725,20 @@ const SORTITION_DB_SCHEMA_4: &'static [&'static str] = &[
 // update this to add new indexes
 const LAST_SORTITION_DB_INDEX: &'static str = "index_burn_header_hash_pox_valid";
 
+const SORTITION_DB_SCHEMA_5: &'static [&'static str] = &[r#"
+    CREATE TABLE peg_in (
+        txid TEXT NOT NULL,
+        vtxindex INTEGER NOT NULL,
+        block_height INTEGER NOT NULL,
+        burn_header_hash TEXT NOT NULL,
+
+        address TEXT NOT NULL,
```

Comment:
What's `address` here again?  Can you add a (sql) comment?  Thanks!

Reply:
Sure, it's the stacks address to receive the `sBTC`. I can add a comment.

Hmm, I think `recipient` would be a better field name throughout the code. I'll rename the field and add a SQL comment for it. Let me know if you have an even better name in mind.

Reply:
✅ Done.